code/__HELPERS/unsorted.dm

```diff
@@ -359,7 +359,7 @@ Turf and target are separate in case you want to teleport some distance from a t
 		x = world.maxx
 	else if(direction & WEST)
 		x = 1
-	if(direction in GLOB.diagonals) //let's make sure it's accurately-placed for diagonals
+	if(ISDIAGONALDIR(direction)) //let's make sure it's accurately-placed for diagonals
```

Comment:
If you could atomize this change, we'd be able to merge it rather fast and it'd diminish the number of changed files here.